cmp/ag-grid/AgGrid.scss

```diff
@@ -121,12 +125,14 @@
 
   &--no-row-borders {
     .ag-row {
-      border-color: transparent;
+      border-bottom: none;
+      border-top: none;
```

Comment:
Removing the border (rather than making it transparent) has the unfortunate side-effect of causing the grid to shift slightly when toggling borders, which does not happen in current toolbox prod.

Not sure how much of an issue that is in practice (border flags are typically not dynamic), but if we can achieve this fix with transparent borders that would be preferable. Something like `border-bottom-color: transparent` etc